simde/simde-common.h

```diff
@@ -1012,9 +1065,13 @@ HEDLEY_DIAGNOSTIC_POP
        || (HEDLEY_GCC_VERSION_CHECK(9,5,0) && !(HEDLEY_GCC_VERSION_CHECK(10,0,0))))
 #      define SIMDE_BUG_GCC_105339
 #    endif
+#    if defined(SIMDE_ARM_NEON_A32V7_NATIVE) && !defined(SIMDE_ARM_NEON_A32V8_NATIVE)
+#      define SIMDE_BUG_GCC_A32V7_MISSFUNC
```

Comment:
Please file a bug with GCC and change this to refer to that bug number

Reply:
This has been deleted since it is the same as `SIMDE_BUG_GCC_95399`.